.github/workflows/codeql.yml

```diff
@@ -13,9 +13,9 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "rjsf-v6" ]
   pull_request:
-    branches: [ "main" ]
+    branches: [ "main", "rjsf-v6" ]
```

Comment:
You can revert these changes now that the `rjsf-v6` branch is merged and deleted